### PR TITLE
TIMOB-19031 Windows: Update Ti.version and Ti.buildDate properties on CI build

### PR DESCRIPTION
**Pull request by sgtcoolguy**
Add build step to replace hard-coded placeholder values in TiModule for git hash and build date

### CODE REVIEW

**Comment by sgtcoolguy** [comment]
test this please.

File: Tools/Scripts/build/build.js [hunk 1]
```diff
@@ -4,13 +4,46 @@
  * Please see the LICENSE included with this distribution for details.
  */
 var path = require('path'),
-	fs = require('fs'),
```
Comment (dawsontoth) [inline Tools/Scripts/build/build.js]: Did you change to two spaced tabbing, Chris?

Comment (dawsontoth) [inline Tools/Scripts/build/build.js]: http://docs.appcelerator.com/platform/latest/#!/guide/JavaScript_Coding_Standards-section-30869180_JavaScriptCodingStandards-Whitespace

Comment (dawsontoth) [inline Tools/Scripts/build/build.js]: (this change would really make the parallelize branch difficult to reconcile)

**Comment by sgtcoolguy** [comment]
@dawsontoth I fixed the indent space/tab issue, this latest should work.

**Comment by dawsontoth** [comment]
so it's not a wip anymore?

**Comment by sgtcoolguy** [comment]
@dawsontoth Oops, yep. changing title now.

**Comment by dawsontoth** [comment]
```javascript
var win = Ti.UI.createWindow({
	backgroundColor: '#fff'
});
win.add(Ti.UI.createLabel({
	text: 'Ti.version: ' + Ti.version 
	+ '\nTi.buildDate : ' 
	+ Ti.buildDate + '\nTi.buildHash : ' + Ti.buildHash,
	textAlign: 'center',
	color: '#000'
}));
win.open();
```

**Pull request merged**